app/mailers/deputy_assignments_mailer.rb

```diff
@@ -2,17 +2,17 @@
 
 class DeputyAssignmentsMailer < ApplicationMailer
   def deputy_assignment_confirmation(deputy_assignment)
-    @primary = deputy_assignment.primary
+    @primary = UserProfile.new(deputy_assignment.primary)
```

Comment:
Just a question so I understand - why are we wrapping the primary user in a user profile here, but not the deputy? And in some other instances, wrapping the deputy, but not primary? Is that just so we have access to the `email` field?

Reply:
Correct, the _only_ thing `UserProfile` does for us is add an `email` method. I could have wrapped them both I suppose.